src/main/java/no/nav/pto/veilarbportefolje/cv/CVRepositoryV2.java

```diff
@@ -69,19 +67,4 @@ public int resetHarDeltCV(AktorId aktoerId) {
         return db.update(updateSql, false, aktoerId.get());
     }
 
-    public Set<AktorId> hentAlleBrukereSomHarCV() {
```

Comment:
why to remove those functions...maybe they are useful :)

Reply:
We can still do this directly in the DB 👍